core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -5580,6 +5586,49 @@ private Sql checkSubQuery(String sql) {
         .withPre(preProgram).with(program).check();
   }
 
+  @Test public void testMonotonicityUDF() throws Exception {
```

Comment:
Can we add a comment like "fix [CALCITE-xxx] link" ?

Reply:
OK